doc/source/examples.rst

```diff
@@ -103,3 +103,77 @@ By default, ``frame_to_hyper`` and ``frames_to_hyper`` will fully drop and reloa
    pantab.frame_to_hyper(df, "example.hyper", table="animals", table_mode="a")
 
 Please note that ``table_mode="a"`` will create the table(s) if they do not already exist.
+
+Inserting New Data into Existing Tables
+---------------------------------------
+
+In addition to append, ``table_mode="i"`` can be used to combine a df with an existing table, only inserting new records. A unique key ``table_key="column"`` must be supplied when using this method.
+
+.. code-block:: python
+
+   import pandas as pd
+   import pantab
+
+   df = pd.DataFrame([
+       ["dog", 4],
+       ["cat", 4],
+   ], columns=["animal", "num_of_legs"])
+
+   # Writing out first df
+   pantab.frame_to_hyper(df, "example.hyper", table="animals")
+
+   updated_df = pd.DataFrame([
+       ["dog", 4],
+       ["cat", 4],
+       ["snake", 0],
+   ], columns=["animal", "num_of_legs"])
+
+   # In this case, only snake will be inserted into your existing table.
+   pantab.frame_to_hyper(updated_df, "example.hyper", table="animals", table_mode="i", table_key="animal")
```

Comment:
From an API perspective I think `table_key` should always be a Sequence; it's quite reasonable for people to use multiple columns as a "primary key"